frontend/src/components/common/Header.svelte

```diff
@@ -3,20 +3,15 @@
   import { page } from '$app/stores';
   import { to } from '$lib/routes';
   import { _ } from 'svelte-i18n';
-  import { locale } from 'svelte-i18n';
   import { Transition } from '@rgossiaux/svelte-headlessui';
   import user from '$stores/user';
   import { logOut } from '$utils/session';
-
   import * as Profile from './Header/Profile';
   import type { UserMenuItem } from './Header/Profile/Desktop.svelte';
   import AuthenticationButtons from './Header/AuthenticationButtons.svelte';
+  import Langswitcher from './Langswitcher.svelte';
 
-  $: if ($locale == 'en-GB') {
-    $locale = 'en';
-  }
-
-  const menuItems = user.isAuthorized
+  $: menuItems = user.isAuthorized
```

Comment:
Are you sure that we need reactive variable here?

Reply:
@LorDEvilDead But you did change it from const to reactive let. Why?

Reply:
Please, leave some reasonable comment about this particular change in Github PR comments or commit message

Reply:
@npupko @MikhailZaitsau done